### PR TITLE
test: Add new initial screen E2E tests

### PR DESCRIPTION
**Pull request by rahulnr7**
1) changes to e2e tests for the new initial and language selection screens

### CODE REVIEW

**Review by Nebyt — APPROVED** [review]

File: apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  it('should be able to verify the ToS link', async () => {
+    await expect(initialScreen.linkToS()).toBeVisible()
+    await initialScreen.linkToS().tap()
+    await expect(element(by.text('1. Rights and Obligations'))).toBeVisible()
+    await utils.takeScreenshot('Terms of Service is displayed')
+    await initialScreen.buttonBack2().tap()
+  })
+
+  it(' should be able to select checkbox and proceed', async () => {
```
Comment (michalszorad) [inline apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts]: There seems to be extra space. Same applies in iOS version

File: apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    await initialScreen.checkboxSelect().tap({x: 5, y: 10})
+    await initialScreen.buttonContinue().tap()
+    await utils.takeScreenshot('User consent screen for sharing insights')
+  })
+
+  it(' should be able to skip consent for analytics and proceed', async () => {
```
Comment (michalszorad) [inline apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts]: There seems to be extra space. Same applies in iOS version

File: apps/wallet-mobile/package.json [hunk 3]
```diff
@@ -21,9 +21,9 @@
     "e2e:test:ios:yoroi:debug": "detox test $npm_package_e2ePath_ios --configuration ios.sim.yoroi.debug",
     "i18n:missed": "i18n-unused display-missed",
     "i18n:unused": "i18n-unused display-unused",
-    "lint": "yarn lint:typescript && prettylint \"src/**/*.tsx\" \"src/**/*.ts\" --config .prettierrc",
-    "lint:fix": "yarn lint:typescript --fix",
```
Comment (michalszorad) [inline apps/wallet-mobile/package.json:25]: Why is the `:typescript` removed? Removing the `:typescript` causes the linter to not fix the files for me

File: apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts (hunk 1 above)
Comment (rahulnr7) [reply apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts]: fixed

File: apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts (hunk 2 above)
Comment (rahulnr7) [reply apps/wallet-mobile/e2e/tests/_android/general-checks.test.ts]: fixed

File: apps/wallet-mobile/package.json (hunk 3 above)
Comment (rahulnr7) [reply apps/wallet-mobile/package.json:25]: changed as discussed. Thanks!

**Review by michalszorad — APPROVED** [review]

**Comment by michalszorad** [comment]
@stackchain should this still be in the 4.20.0 milestone?

**Comment by stackchain** [comment]
Iv'e added

**Pull request merged**